mm2src/mm2_main/tests/docker_tests/qrc20_tests.rs

```diff
@@ -1064,6 +1064,7 @@ fn test_get_max_taker_vol_and_trade_with_dynamic_trade_fee(coin: QtumCoin, priv_
     // where `available = balance - locked_amount - max_trade_fee - max_fee_to_send_taker_fee`
     let available = &qtum_balance - &max_trade_fee - &max_fee_to_send_taker_fee;
     debug!("total_available: {}", available);
+    #[allow(clippy::redundant_clone)] // This is a false-possitive bug from clippy
     let min_tx_amount = qtum_min_tx_amount.clone();
     let expected_max_taker_vol =
         max_taker_vol_from_available(MmNumber::from(available), "QTUM", "MYCOIN", &min_tx_amount)
```

Comment:
```suggestion
    debug!("total_available: {}", available);
    let expected_max_taker_vol =
        max_taker_vol_from_available(MmNumber::from(available), "QTUM", "MYCOIN", &qtum_min_tx_amount)
            .expect("max_taker_vol_from_available");
```

Reply:
I kind of liked the way we explicitly defined the minimum transaction amount. The change in suggestion is not really clear at first glance (to figure out the min tx amount, we need to look at `max_taker_vol_from_available` and check what the last argument stands for). Considering that this is just a test function, I am not sure if this change is worth applying. These are my thoughts, if you still think we should apply this change, please feel completely free to do that!